src/main/java/core/basesyntax/figures/Circle.java

```diff
@@ -0,0 +1,35 @@
+package core.basesyntax.figures;
+
+import core.basesyntax.Figure;
+
+public class Circle extends Figure {
```

Comment:
The class Circle should not begin with an empty line according to the checklist.